pyarrow/build.sh

```diff
@@ -0,0 +1,45 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Should add `# *****************************************************************` above the comments.